### PR TITLE
Task category ordering

### LINKED ISSUE #203: Tasks category ordering
# Requirement
- Have `Active` category first and keep it open
- Order doesn't matter for the rest of the categories
- Keep the rest of the categories closed

### PR DESCRIPTION
**Pull request by shubham-y**
Closes #203

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -23,6 +30,8 @@ const Index: FC = () => {
     if ('tasks' in response) {
       tasks = response.tasks;
       tasks.sort((a:task, b:task) => +a.endsOn - +b.endsOn);
+      const statusOrder = [ACTIVE, ASSIGNED, COMPLETED, PENDING, UNASSIGNED];
```
Comment (whyDontI) [inline src/pages/index.tsx:33]: Please move this on line no: 18 as this value is known prior to execution.
Also, please make the name STATUS_ORDER.
https://javascript.info/variables#uppercase-constants

**Review by whyDontI — CHANGES_REQUESTED** [review]
Good work 🥳 
Requesting a small change

File: src/pages/index.tsx (hunk 1 above)
Comment (shubham-y) [reply src/pages/index.tsx]: Made the requested changes

**Review by whyDontI — APPROVED** [review]
Good work 🥳

**Pull request merged**